src/panels/lovelace/cards/hui-picture-elements-card.js

```diff
@@ -30,19 +30,17 @@ class HuiPictureElementsCard extends LocalizeMixin(EventsMixin(PolymerElement))
       #root img {
         width: 100%;
       }
-      #root .entity {
+      .entity {
+        line-height: 1em;
```

Comment:
Why is this?

Reply:
in ha-card I use `line-height: 0` to avoid a gap at the bottom of the card.

I only "need" this for the div with state-text to show the correct element size

Reply:
I think that you should be able to achieve anything without the `line-height: 0`. Let's adjust the template?

Reply:
can you help? we talk about these 6px
![unbenannt](https://user-images.githubusercontent.com/11984118/41983045-4296668e-7a2d-11e8-8b73-af646eb412bc.png)

Reply:
ok